samples/zigbee/ncp/README.rst

```diff
@@ -177,27 +176,22 @@ After building the sample and programming it to your development kit, test it by
 1. Download and extract the `ZBOSS NCP Host`_ package.
 
    .. note::
-      If you are using an Linux distribution different than the 64-bit Ubuntu 18.04, make sure to rebuild the package libraries and applications by following the instructions described in the :file:`README.rst`  file in the ZBOSS NCP Host package.
+      If you are using an Linux distribution different than the 64-bit Ubuntu 18.04, make sure to rebuild the package libraries and applications by following the instructions described in the :file:`README.rst` file in the ZBOSS NCP Host package.
 
 #. If you are using `Communication through USB`_, connect the nRF USB port of the NCP sample's development kit to the PC USB port with an USB cable.
 #. Get the kit’s serial port name (for example, /dev/ttyACM0).
    If you are communicating through the nRF USB, get the nRF USB serial port name.
-#. Turn on the development kit that runs the network coordinator sample.
-   When **LED 3** turns on, this development kit has become the Coordinator of the Zigbee network.
 #. Turn on the development kit that runs the light bulb sample.
-   When **LED 3** turns on, the light bulb has become a Router inside the network.
-
-   .. tip::
-        If **LED 3** does not turn on, press **Button 1** on the Coordinator to reopen the network.
-
-#. Start the light control application by running the following command with *serial_port_name* replaced with the serial port name used for communication with the NCP sample:
+#. Start the simple gateway application by running the following command with *serial_port_name* replaced with the serial port name used for communication with the NCP sample:
 
    .. parsed-literal::
       :class: highlight
 
-      NCP_SLAVE_PTY=*serial_port_name* ./application/light_sample/light_control/light_control
+      NCP_SLAVE_PTY=*serial_port_name* ./application/simple_gw/simple_gw
 
-The light control device joins the Zigbee network, searches for a light bulb to control, and starts sending On/Off commands with a 15-sec period that toggle the **LED 4** on the light bulb on and off.
+The simple gateway device joins the Zigbee network and searches for a light bulb to control.
```

Comment:
The simple gateway forms Zigbee network, not joins.

Better sentences would be:
"simple gateway device forms Zigbee network and opens the network for 180s for new devices. When light bulb joins, the LED 3 on the light bulb device turns on to indicate that it is connected to the simple gateway. The gateway then starts discovering On/Off cluster. If it is found, Simple gateway configures bindings and reporting for the device and
then it starts sending On/Off toggle commands with a 15-sec period that toggle the LED 4 on the light bulb on and off"

or something like that